### PR TITLE
Add idr0141 to studies.tsv

### PR DESCRIPTION
**Pull request by will-moore**
cc @francesw

### CODE REVIEW

**Comment by sbesson** [comment]
```
* At /site/about/screens.html:288:


  External link https://github.com/IDR/idr-metadata/tree/master/idr0006-fong-nuclearbodies failed (status code 404)

* At /site/about/submission.html:209:

  External link https://github.com/IDR/idr-metadata/tree/master/idr0006-fong-nuclearbodies failed (status code 404)
 ```

@francesw has extracted these metadata files to a standalone repository - see https://github.com/IDR/idr0006-fong-nuclearbodies

**Review by sbesson — CHANGES_REQUESTED** [review]
Looks good. Since `idr0141` is the only released study, can you also update `releases.tsv` for `prod110`?

**Comment by will-moore** [comment]
@sbesson releases.tsv done

File: _data/releases.tsv [hunk 1]
```diff
@@ -63,3 +63,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2022-07-05	prod107	0.10.7	20079	1857867		13082709	107502723	319.155132865449	32.051303	590.240281247	110
 2022-07-25	prod108	0.10.8	20086	1858535		13108079	108358263	320.288450979338	32.102065	591.651451551	111
 2022-08-25	prod109	0.10.9	20182	1858535		13113214	109031541	335.244769283323	32.117471	592.08	112
+2022-09-29	prod110	0.11.0	20184	1858535		13113224	109031551	335.25245153854803	32.117502	546.707092127	113
```
Comment (sbesson) [inline _data/releases.tsv:66]: Not that I am particular concerned by the DB size in general but a decrease would typically happen when we perform operations such as dump/restore as this also makes optimisations and reduces space.

I don't recall doing any such thing for `prod110`. How was the DB size computed?

Comment (will-moore) [reply _data/releases.tsv:66]: Ah - my bad.
It seems I started in a console that was already logged-in to idr0125-pilot and didn't notice that the login to idr-next failed....

```
(base) [wmoore@pilot-idr0125-omeroreadwrite ~]$ ssh idr-next.openmicroscopy.org -At ssh omeroreadwrite -A
ssh: Could not resolve hostname idr-next.openmicroscopy.org: Name or service not known
(base) [wmoore@pilot-idr0125-omeroreadwrite ~]$ export PGPASSWORD=$(/opt/omero/server/OMERO.server/bin/omero config get omero.db.pass)
(base) [wmoore@pilot-idr0125-omeroreadwrite ~]$ echo $PGPASSWORD 
2NGS5rrKNqsnR6sasDLa71CG+IpoBmIoQwEVXarc2cto
(base) [wmoore@pilot-idr0125-omeroreadwrite ~]$ export PGHOST=$(/opt/omero/server/OMERO.server/bin/omero config get omero.db.host)
(base) [wmoore@pilot-idr0125-omeroreadwrite ~]$ psql -U omero idr -c "select pg_database_size('idr')"
 pg_database_size 
------------------
     546707092127
(1 row)
```
I'll run again...

**Review by sbesson — APPROVED** [review]

**Pull request merged**